nss-day-cohort-48/giffygram-altavista-giffygram
# Footer Implementation

#### Implemented The Footer



1. Added file `Footer.js` to `./src/scripts/nav/Footer.js` directory.
1. Modified file `Footer.js` to include `export footer and change events` and functionality.
   
   ​

#### Steps to Review

1. Checkout this branch locally.
   ```
   git fetch --all
   git checkout ken-giffygram
   ```
2. Open a new Terminal and open the giffygram repo (make sure it's named 'giffygram')
3. Run `./serve-giffygram` to serve the code.
4. Test app functionality.
   > Test footer functionality
   > Make sure all the filters are working properly
5. View code file.
   > Confirm file modifications are present as indicated above.
   > Confirm no unused code or extraneous comments exist.
